app/views/govuk_publishing_components/components/_published_dates.html.erb

```diff
@@ -0,0 +1,50 @@
+<%
+  add_gem_component_stylesheet("published-dates")
+
+  published ||= false
+  history ||= []
+  history = Array(history)
+  last_updated ||= false
+  link_to_history ||= false
+
+  component_helper = GovukPublishingComponents::Presenters::ComponentWrapperHelper.new(local_assigns)
+  component_helper.add_class("gem-c-published-dates")
+  component_helper.add_class("gem-c-published-dates--history") if history.any?
+  component_helper.set_id("full-publication-update-history") if history.any?
+  component_helper.add_data_attribute({ module: "gem-toggle" }) if history.any?
+  component_helper.set_lang("en")
+%>
+<% if published || last_updated %>
+  <%= tag.div(**component_helper.all_attributes) do %>
+    <h2 class="govuk-visually-hidden"><%= t('components.published_dates.hidden_heading') %></h2>
+    <% if published %>
+      <%= t('components.published_dates.published', date: published) %>
+    <% end %>
+    <% if last_updated %>
+      <% if published %><br /><% end %><%= t('components.published_dates.last_updated', date: last_updated) %>
```

Comment:
Not for this PR, but we should maybe circle back and look at why this `<br/>` is being inserted and whether it could be resolved with CSS instead.

Reply:
Looking through the rest of this file there's some similar things that perhaps could be done better, like the non breaking space on line 36 and other HTML symbols e.g. line 32.

Reply:
Thanks - will raise an issue for this 👍